src/utils/auth_utils.jsx

```diff
@@ -0,0 +1,413 @@
+/* eslint-disable no-console */
+/* eslint-disable react/react-in-jsx-scope */
+import axios from 'axios';
+import { v4 as uuidv4 } from 'uuid';
+import moment from 'moment';
+
+import { initializeApp } from 'firebase/app';
+
+import {
+  getAuth,
+  signInWithEmailAndPassword,
+  signOut,
+  GoogleAuthProvider,
+  signInWithPopup,
+  getAdditionalUserInfo,
+  createUserWithEmailAndPassword,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'createUserWithEmailAndPassword' is defined but never used.